src/signature_pad.ts

```diff
@@ -37,13 +37,57 @@ export interface Options extends Partial<PointGroupOptions> {
   velocityFilterWeight?: number;
   backgroundColor?: string;
   throttle?: number;
+  documentAsEventTarget?: boolean;
 }
 
 export interface PointGroup extends PointGroupOptions {
   points: BasicPoint[];
 }
 
-export default class SignaturePad extends EventTarget {
+class SignatureEventTarget implements EventTarget {
```

Comment:
this class should move to it's own file.

Reply:
noted. i'm happy to change this in my next commit.

Reply:
hey since you're currently online, do you prefer 'signature-event-target.ts' or 'signature_event_target.ts' ? What's the convention in this project?